app/controllers/articles_controller.rb

```diff
@@ -28,7 +29,10 @@ def create
 
     respond_to do |format|
       if @article.save
-        format.html { redirect_to @article, notice: 'Article was successfully created.' }
+        format.html {
+          redirect_to @article, notice: 'Article was successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.